PostgreSQL/OMOP CDM comments - PostgreSQL.sql

```diff
@@ -0,0 +1,481 @@
+--
+--CLINICAL
```

Comment:
These comments are great but can you also provide a code to retrieve them in the [CodeExcerpts](https://github.com/OHDSI/CommonDataModel/tree/master/CodeExcerpts) table?